app/models/concerns/unparsed_run.rb

```diff
@@ -58,10 +59,13 @@ def parse_into_db
 
           total_playtime_ms:       run_data[:total_playtime_ms],
           default_timing:          run_data[:default_timing],
-          filesize_bytes:          run_data[:size]
+          filesize_bytes:          run_data[:size],
+
+          platform:                SpeedrunDotComPlatform.find_by(name: run_data[:metadata][:platform_name]),
+          region:                  SpeedrunDotComRegion.find_by(name: run_data[:metadata][:region_name])
         )
 
-        HighlightSuggestion.from_run(self)
+        # HighlightSuggestion.from_run(self)
```

Comment:
Was this left accidentally?

Reply:
Yep, probably throwing an error so I fixed it the old fashioned way.